airflow/utils/log/file_task_handler.py

```diff
@@ -150,8 +151,16 @@ def read(self, task_instance, try_number=None):
             try_numbers = [try_number]
 
         logs = [''] * len(try_numbers)
-        for i, try_number in enumerate(try_numbers):
-            logs[i] += self._read(task_instance, try_number)
+
+
+        try:
+            for i, try_number in enumerate(try_numbers):
+                logs[i] += self._read(task_instance, try_number)
+
+        except TypeError as error:
```

Comment:
Where is the TypeError coming from?

Reply:
NoneType if there are no logs available.

Reply:
I found this while running a test dag in airflow.

Reply:
Huh.  Okay cool.